auto_gptq/utils/marlin_utils.py

```diff
@@ -93,33 +95,33 @@ def _get_cached_marlin_save_name(model_name_or_path):
     If local:    {model_name_or_path}/autogptq_model.safetensors
     """
     if os.path.isdir(model_name_or_path):
-        return os.path.join(model_name_or_path, "autogptq_model.safetensors")
+        return os.path.join(model_name_or_path, MARLIN_CACHE_FILE)
     else:
         namespace, subfolder = model_name_or_path.split("/")
         assets_path = huggingface_hub.cached_assets_path(
             library_name="auto_gptq", namespace=namespace, subfolder=subfolder
         )
-        return os.path.join(assets_path, "autogptq_model.safetensors")
+        return os.path.join(assets_path, MARLIN_CACHE_FILE)
 
 
-# Validate marlin suppor
+# Validate marlin support
 def _validate_marlin_device_support():
-    device_capacity = torch.cuda.get_device_capability()
-    return device_capacity[0] == 8 and (device_capacity[1] == 0 or device_capacity[1] == 6)
-
+    return torch.cuda.get_device_capability()[0] >= 8
```

Comment:
@fxmarty  latest code added separation of check between marlin_supported and marlin_optimized. logging.info added when marlin_optimized is false. I don't think it warrants a warning level.

My understand of following req is that hooper is not fully optimized in Marlin for maximum performance but still perfomant vs ada/ampere since the marlin kernel is optimized for ada/ampere sm cores/cache sizes. Everything ada/amper has hopper does too: just that hopper will be under-utilized. I don't have hopper to test though.

```
NVIDIA GPU with compute capability >= 8.0 (Ampere or Ada, Marlin is not yet optimized for Hopper)
```